bin/install

```diff
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-# shellcheck source=../lib/utils.sh
+# shellcheck source=/dev/null
```

Comment:
https://www.shellcheck.net/wiki/SC1090
https://www.shellcheck.net/wiki/SC1091
https://github.com/koalaman/shellcheck/wiki/directive

I have tried `source-path` directive. However it still made alerts. 🤔